src/parser/priest/holy/modules/spells/Renew.js

```diff
@@ -110,6 +147,61 @@ class Renew extends Analyzer {
       this.renewsFromBenedictionAndRenew++;
     }
   }
+
+  on_byPlayer_globalcooldown(event) {
+    const spellId = event.ability.guid;
+    if (spellId !== SPELLS.RENEW.id) {
+      return;
+    }
+    this.lastGCD = event;
+  }
+
+  // This function validates if a renew should have been cast at all.
+  validateRenew(event) {
+    this.lastRenewCast = event.timestamp;
+    if (this.lastGCD && this.movedSinceCast(event)) {
+      // We are moving, but do we have any other instant cast spells?
+      const sanctifyOnCooldown = this.spellUsable.isOnCooldown(SPELLS.HOLY_WORD_SANCTIFY.id);
+      const serenityOnCooldown = this.spellUsable.isOnCooldown(SPELLS.HOLY_WORD_SERENITY.id);
+      let cohOnCooldown = true;
+      if (this.selectedCombatant.hasTalent(SPELLS.CIRCLE_OF_HEALING_TALENT.id)) {
+        cohOnCooldown = this.spellUsable.isOnCooldown(SPELLS.CIRCLE_OF_HEALING_TALENT.id);
+      }
+      if (sanctifyOnCooldown && serenityOnCooldown && cohOnCooldown) {
+        this.goodRenews += 1;
+      } else {
+        this.badRenewReason.betterspell = (this.badRenewReason.betterspell | 0) + 1;
```

Comment:
This should be `||` right? unless there is a reason to use bitwise or that I'm unaware of?

Reply:
Fixed